### PR TITLE
Add the new python-planarity dependency including windows

### PR DESCRIPTION
**Pull request by gonzalocasas**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,9 +15,9 @@ source:
 
 build:
   skip: True   # [linux and py27]
-  number: 0
+  number: 1
   script:
-    - {{ PYTHON }} -m pip install --upgrade pip  # [win]
+    - {{ PYTHON }} -m pip install --upgrade pip
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]: why do you need to do this?

Comment (gonzalocasas) [reply recipe/meta.yaml]: you mean update pip at all? I don't really remember, I'll get rid of it.

Comment (mariusvniekerk) [reply recipe/meta.yaml]: using pip to install anything other than the package you're building  will include things you don't want in your package

Comment (gonzalocasas) [reply recipe/meta.yaml]: thanks, it's removed now 👍

**Pull request merged**